gating/gate/run

```diff
@@ -26,4 +26,28 @@
 # trigger.
 # RE_JOB_REPO_NAME: The project's GitHub repository which gets cloned in the
 # job. For example, for the repository found at
-# https://github.com/rcbops/rpc-gating, the name is rpc-gating.
\ No newline at end of file
+# https://github.com/rcbops/rpc-gating, the name is rpc-gating.
+
+set -eux
+set -o pipefail
+
+cd /opt/magnanimous-turbo-chainsaw/scripts/
+bash setup.sh
+
+# Ensure the deployment workspace is loaded and ready to run workloads
+. setup-workspace.sh
+
+# Probe the environment for any host that is unreadchable
+cd /opt/magnanimous-turbo-chainsaw/playbooks/
+ansible-playbook probe-systems.yml
+
+# Clean up an existing ELK environment
+cd /opt/magnanimous-turbo-chainsaw/
+. ./scripts/cleanup-legacy-elk.sh
+
+# deactivate workspace
+deactivate
```

Comment:
this should be above the clean-up script too.

Reply:
Done